### ISSUE TITLE
FAILURE: Build failed with an exception

### ISSUE THREAD

**Issue by otorenbosch** [facebook/react-native#29703]
Please provide all the information requested. Issues that do not follow this format are likely to stall.

## Description
FAILURE: Build failed with an exception.

* Where:
Script 'C:\Users\Temp\zoekapp\node_modules\react-native\react.gradle' line: 238

* What went wrong:
A problem occurred configuring project ':app'.
> Could not get unknown property 'mergeResourcesProvider' for object of type com.android.build.gradle.internal.api.ApplicationVariantImpl.

## React Native version:
System:
    OS: Windows 10 10.0.18363
    CPU: (4) x64 Intel(R) Core(TM) i5-7300U CPU @ 2.60GHz
    Memory: 3.94 GB / 7.92 GB
  Binaries:
    Node: 12.18.3 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 6.14.6 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK:
      API Levels: 25, 26, 29, 30
      Build Tools: 26.0.2, 27.0.3, 28.0.3, 29.0.2, 30.0.2
      System Images: android-26 | Intel x86 Atom_64, android-26 | Google APIs Intel x86 Atom, android-29 | Intel x86 Atom_64, android-29 | Google APIs Intel x86 Atom_64, android-30 | Google APIs Intel x86 Atom, android-30 | Google APIs Intel x86 Atom_64
      Android NDK: Not Found
    Windows SDK: Not Found
  IDEs:
    Android Studio: Version  4.0.0.0 AI-193.6911.18.40.6626763
    Visual Studio: Not Found
  Languages:
    Java: 1.8.0_262
    Python: 2.7.18
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.0.0-alpha.12 => 16.0.0-alpha.12
    react-native: ^0.63.2 => 0.63.2
  npmGlobalPackages:
    *react-native*: Not Found

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. Existing project from someone else
2. Did the installation of React  through https://reactnative.dev/docs/environment-setup
3. In the folder I did: npm install
4. After that i did: npm audit-fix
5. Then npx react-native run-android
6. At the build the error message

## Expected Results
I expect that my app would build and that i would see my app in an android emulator

**Comment by ivanmoskalev** [comment]
Could you please post here contents of the following files:
1. `android/build.gradle`
2. `android/gradle/wrapper/gradle-wrapper.properties`

It is likely that the project is using an outdated version of Gradle. You can try to ensure your `android/gradle/wrapper/gradle-wrapper.properties` contains [this](https://github.com/facebook/react-native/blob/master/template/android/gradle/wrapper/gradle-wrapper.properties).

**Comment by otorenbosch** [comment]
Package.json
![image](https://user-images.githubusercontent.com/69941158/90758049-b2498080-e2de-11ea-85ef-29077d60581d.png)

Android/build.gradle
![image](https://user-images.githubusercontent.com/69941158/90758139-d907b700-e2de-11ea-8275-c49e35811bd1.png)

Android/gradle/wrapper/gradle-wrapper.properties
![image](https://user-images.githubusercontent.com/69941158/90758250-06546500-e2df-11ea-845e-effeafe61baa.png)

**Comment by otorenbosch** [comment]
My project is build with react-native 0.48.3 when i try to run-android with this version of react-native i get the following error.
![image](https://user-images.githubusercontent.com/69941158/90760630-77494c00-e2e2-11ea-8e37-25e3559c909c.png)

The suggested solution didnt work. 
I found out that after i did an npm install(0.48.3) there were a lot vulnerabilities and i cannot execute npx react-native run-android

After i did an npm audit fix i saw that my version of react-native has been set to 0.63.2. Now the npx react-native run-android works, but then i will get the error "FAILURE: Build failed with an exception" where i started this issue with.

**Comment by otorenbosch** [comment]
Tonight i got my app working on an virtuel device. What i did to do that.
1. freshly install of the project in a new folder
2. Run "npm install" in a command prompt to install the packages in packages.json
3. Did not fix the vulnerabilities!!
4. Run "npm install react-native-cli"
5. Start a virtuel device in Android Studio
6. Run "npx react-native run-android"

The app crashed when i started it up on my virtuel device. I read somewhere that sometime thats because of the Node js. version, so I deleted V12.18.3 and I installed V12.9.1. Now the app don't crash anymore and everything is working fine!

Now I will try to upgrade the app so it is working with the latest React-native and packages.

If anyone have some help how I should do that, let me know!
